src/incubator/TextField/index.tsx

```diff
@@ -123,7 +124,7 @@ const TextField = (props: InternalTextFieldProps) => {
 
   return (
     <FieldContext.Provider value={context}>
-      <View style={[margins, positionStyle, containerStyle, centeredContainerStyle]}>
+      <View id={containerId} style={[margins, positionStyle, containerStyle, centeredContainerStyle]}>
```

Comment:
I suggest changing it to `containerProps` to allow passing any prop the user wants to the container (of type of `ViewProps` without the style since we have the `containerStyle` prop)